pytalibstream/atr_keeper.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+import pyslidingwindow
+from pytalibstream.types.hlc import HLC
+
+
+class AtrKeeper:
+    def __init__(self, max_len: Optional[int] = 10):
+        if max_len < 2:
+            raise Exception(
+                "AtrKeeper period must be >= 2")
+
+        self.high = pyslidingwindow.SlidingWindowFloat(max_len)
+        self.low = pyslidingwindow.SlidingWindowFloat(max_len)
+        self.close = pyslidingwindow.SlidingWindowFloat(max_len)
+        self._atr = 0
+        self.data_count = 0
+        self.max_len = max_len
+        self.prev_tr = []
+
+    def add(self, candle: HLC):
```

Comment:
I feel like creating a tuple may slow down the process although I do not have clear measurements.
imagine we are gonna need to create tuple in order to call this function for millions of times.
I think it might be safer to make it a simple function taking 3 args.
```def add(self, high: float, low: float, close: float):```